### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by ghost**
Use grunt build and grunt serve before
links : 
localhost/wiki/edit/name.of.the.plugin    -->  for the wiki editor
localhost/name.of.the.plugin/comments  --> for the comments
connected to #89
connected to #88

### CODE REVIEW

File: shuttleofx_client/static_tmp/styles/wiki.less [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#metaWiki fieldset {
+    // border: 1px solid #2ab074;
```
Comment (Olivier-Daire) [inline shuttleofx_client/static_tmp/styles/wiki.less]: Unused code ? Remove it :smile:

Comment (ghost) [inline shuttleofx_client/static_tmp/styles/wiki.less]: thanks :p I'm on it

**Comment by KrisNumber24** [comment]
It works !
But  : 
- in wiki, there is a Preview section at the bottom of the page but it displays nothing. Maybe you could delete this section.
- in comments, the display is strange, you have a list for the name and a list for the text...

**Comment by Olivier-Daire** [comment]
Already said in the wiki branch that has been closed...
- when entering code inside ``` the indentation is not kept, and the syntax highlighting doesn't seems to work
- already said it but the preview button is not very well placed, in my mind, plus I think that there should be a change when you are in preview vs when you are in edit mode
- why is there a `<h3>Preview :</h3>` below the editor ? [EDIT] ok got it, it's when you save, but it think it kind of weird when you edit it
- a small padding on the left between the text and the input border would be great
- fullscreen mode is very cool ! But the preview button text goes black when we go into preview mode
- would be great if we could use `CTRL Z` but this may be complicated

**Comment by ghost** [comment]
For the next update (coming soon)

Good news : 

---
- For the preview, I removed the button, instead we'll have a dynamic preview just below
- For the `<h3>Preview : </h3>`, it was a mistake, now it has been deleted
- A small padding on the left between the text and the border has been set
- Fullscreen mode has been updated
- `CTRL-Z` is already working :p

Bad news : 

---
- For ``` it is unfortunately normal, the developers themselves of Markdown Bootstrap and Markjown to Html do not use ``` for the same reasons,

**Comment by KrisNumber24** [comment]
Maybe you could change the behaviour of the button code in the editing wiki page to add four blank spaces to allow the users to indent their code.
I think you should change style of comments page because it appears really empty.

**Comment by ghost** [comment]
Done !

**Comment by Olivier-Daire** [comment]
Nice ! :+1: 

Few things to improve : 
- **comment page** : change button wording to something more usual "send" or "comment" for example, change button's color
- **comment page** : try to display the date in a better way "16 Jan 2016" for example. At least remove the comma after the date

**Comment by KrisNumber24** [comment]
If it's possible, try to enhance code typing in the wiki page.

**Pull request merged**